subsys/usb/usb_device.c

```diff
@@ -234,7 +234,7 @@ static bool usb_handle_request(struct usb_setup_packet *setup,
  */
 static void usb_data_to_host(void)
 {
-	u32_t chunk = min(MAX_PACKET_SIZE0, usb_dev.data_buf_residue);
+	u32_t chunk = usb_dev.data_buf_residue;
```

Comment:
This change should be done in a separate commit (currently it is in "drivers: usb: Add nRF52840 USBD driver") and described properly in the commit message.

Reply:
Maybe it would be better to open own PR for it, I do not sure that every driver can handle it. We should test existing drivers before it is merged.

Reply:
@jfischer-phytec-iot  @finikorg I think that's probably the safest.
@pawelzadrozniak can you please open a separate PR with this change so we can get that tested and merged separately?